### PR TITLE
[Bgp-speaker]: rm no need code

### PR DESCRIPTION
**Pull request by sihuihan88**

### CODE REVIEW

File: ansible/roles/test/tasks/bgp_speaker.yml [hunk 1]
```diff
@@ -49,9 +49,6 @@
 
 # vlan_ips[0], speaker_ips[0], speaker_ips[1] are IPs for three bgp speakers
 # vlan_ips[1], vlan_ips[2] are IPs for mux under DUT.
```
Comment (stcheng) [inline ansible/roles/test/tasks/bgp_speaker.yml]: are these comments related?

Comment (sihuihan88) [inline ansible/roles/test/tasks/bgp_speaker.yml]: No. They are unrelated
